### DISCUSSION TITLE
Direct messages should never have gone live in their current form

### DISCUSSION THREAD

**Discussion by Voroxpete** [LemmyNet/lemmy-ui#1948] [General]
I get it. Open source is hard. Everyone is donating their time, and the expectation is that features start out kind of basic and improve steadily over time.
However social media comes with its own very specific set of expectations in regard to users safety and wellbeing.
A direct message system that not only allows users to send images and videos directly to each other, but then automatically displays / plays that content is so obviously ripe for horrific levels of abuse that I cannot begin to fathom how it was considered acceptable to merge in the feature in that form.
Right now users of major lemmy instances are opening up their inboxes to find endless videos of people enthusiastically eating shit. That is not a metaphor. And I can only imagine the sea of dick pics that anyone brave enough to openly admit they're female on this platform must have to wade through.
I cannot even begin to imagine why users would need to be able to DM each other images in the first place. There is scarcely any good reason for this option to exist, and certainly no good reason for it to exist in its current form. It would be better to disable direct messaging entirely than to allow such a horrifically abusable feature to be present in a social media platform.

**Comment by techno156** [comment]
Would it make more sense to have this be a bug instead, considering the security risk?

Especially if the image is linked externally.

**Reply by Voroxpete** [reply]
Yes, I completely agree. The current implementation presents a real risk to users.

It's possible that the delete button on its own could be considered a feature, but the ability to send images directly is the real security and safety issue here. Even if it was the intended design at the time, it should be considered a flaw at this point, and one that urgently needs addressing.

Edit to add: Sorry, forget which comment thread I was replying to. The reason I posted this under discussion is because there are already some relevant issues open, though they do seem to be be marked as features rather than bugs.

**Comment by Nolram12345** [comment]
When a user even tries to send another a DM, there is also always that odd banner that just seems to link to the commercial service element.io for no reason... I don't think its appropriate for it to be there considering its not related to Lemmy itself.
